--- origsrc/gmetadom-0.2.6/src/gdome_cpp_smart/Makefile.am	2007-05-08 10:26:17.000000000 -0500
+++ src/gmetadom-0.2.6/src/gdome_cpp_smart/Makefile.am	2007-10-15 21:59:02.375000000 -0500
@@ -4,13 +4,13 @@
 lib_LTLIBRARIES = libgmetadom_gdome_cpp_smart.la
 
 libgmetadom_gdome_cpp_smart_la_LIBADD = \
-  $(GDOME_LIBS) \
   $(top_builddir)/src/gdome_cpp_smart/events/libevents.la \
   $(top_builddir)/src/gdome_cpp_smart/core/libcore.la \
   $(top_builddir)/src/gdome_cpp_smart/basic/libbasic.la \
+  $(GDOME_LIBS) \
   -lstdc++
 
-libgmetadom_gdome_cpp_smart_la_LDFLAGS = -version-info @GMETADOM_VERSION_INFO@
+libgmetadom_gdome_cpp_smart_la_LDFLAGS = -version-info @GMETADOM_VERSION_INFO@ -no-undefined
 
 libgmetadom_gdome_cpp_smart_la_SOURCES = \
   GdomeSmartDOMDOMImplementation.cc
--- origsrc/gmetadom-0.2.6/src/gdome_cpp_smart/test/Makefile.am	2007-05-08 10:26:18.000000000 -0500
+++ src/gmetadom-0.2.6/src/gdome_cpp_smart/test/Makefile.am	2007-10-15 21:59:02.609375000 -0500
@@ -6,9 +6,8 @@
 deep_SOURCES = deep.cc
 
 LDADDS = \
-  $(GDOME_LIBS) \
-  $(top_builddir)/src/gdome_cpp_smart/.libs/libgmetadom_gdome_cpp_smart.a \
   $(top_builddir)/src/gdome_cpp_smart/libgmetadom_gdome_cpp_smart.la \
+  $(GDOME_LIBS) \
   -lstdc++
 
 test_LDADD = $(LDADDS)
