--- origsrc/gmetadom-0.2.6/src/gdome_caml/Makefile.am	2006-08-06 03:33:36.000000000 -0500
+++ src/gmetadom-0.2.6/src/gdome_caml/Makefile.am	2007-10-15 21:59:02.359375000 -0500
@@ -48,10 +48,10 @@
 	$(patsubst %, events/%, $(shell cat $(srcdir)/events/.linkorder))	\
 	$(shell cat $(srcdir)/.linkorder)
 O_S =	\
-	  $(patsubst basic/%.c, basic/.libs/%.o, $(wildcard basic/ml_*.c))	\
-	  $(patsubst core/%.c, core/.libs/%.o, $(wildcard core/ml_*.c))	\
-	  $(patsubst events/%.c, events/.libs/%.o, $(wildcard events/ml_*.c))	\
-	  $(patsubst %.c, .libs/%.o, $(wildcard ml_*.c))
+	  $(wildcard basic/.libs/ml_*.o) \
+	  $(wildcard core/.libs/ml_*.o) \
+	  $(wildcard events/.libs/ml_*.o) \
+	  $(wildcard .libs/ml_*.o)
 LO_S = $(patsubst %.o, %.lo, $(O_S))
 CMA_DEPS = $(LIB_DEPS:%=%.cmo)
 CMXA_DEPS = $(LIB_DEPS:%=%.cmx)
--- origsrc/gmetadom-0.2.6/src/gdome_caml/include/Makefile.am	2002-08-30 13:03:36.000000000 -0500
+++ src/gmetadom-0.2.6/src/gdome_caml/include/Makefile.am	2007-10-15 21:59:02.375000000 -0500
@@ -4,5 +4,5 @@
 
 install-data-local: mlgdomevalue.h
 	$(mkinstalldirs) $(OCAMLINSTALLDIR)
-	$(INSTALL_DATA) $(srcdir)/$< $(OCAMLINSTALLDIR);
+	$(INSTALL_DATA) $< $(OCAMLINSTALLDIR);
 
